website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -111,6 +111,19 @@ and [after](https://github.com/hashicorp/consul/blob/71d45a34601423abdfc0a64d44c
 
 1. (Optionally) upgrade Envoy to the latest version supported in Consul 1.10.0.
 
+### Transparent Proxy on Kubernetes
+
+When upgrading to Consul >= 1.10.0 and Consul-helm >= 0.32.0 and Consul-k8s >= 0.26.0, a change will be necessary for existing services:
+
+- A Kubernetes Service must be added for every service registered to Consul. This Service should be added before
+performing the upgrade. This will allow services to be managed by a central component which will enable features like
+transparent proxy.
+
+After the upgrade is performed, all Pods of a service will need to be restarted. The service will be up and health
+checks will continue to work without restarting the service, but it is strongly recommended to restart the Pods for the
+new service management mechanism to take effect.
```

Comment:
I wonder if we can link to a k8s doc about rollout upgrades here? Does that work in our current upgrade path?